content/en/docs/contribute/style/style-guide.md

```diff
@@ -358,6 +359,70 @@ Shortcodes inside include statements will break the build. You must insert them
 {{</* include "task-tutorial-prereqs.md" */>}}
 {{</* /note */>}}
 ```
+## Markdown elements
+
+
+### Front matter
+Hugo uses front matter information to create things like author attribution, categories, and page layout. Copying an existing file, then updating its filename and front matter to can be an easy way to help ensure everything is formatted properly. 
+
+### Line breaks
+Use a single newline to separate block-level content like headings, lists, images, code blocks, and others. The exception is second-level headings, where it should be two newlines. This helps visualize the overall structure of content in a code editor.
+
+### Headings
+Remember that screen readers are linear output devices which means they output items on a page one at a time, so if there is a lot of content on your page, applying headings gives the page a structure that allows the user to easily navigate, without having to read loads of content that is of no use, value, or interest to them.
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td><ul><li>Use ordered headings to provide a meaningful high-level outline of your content.</li>
+  <li>Use only one first-level heading per page.</li>
+  <li>For non-blog post content, use pound or hash signs (#), not underlines (--- or ===) to designate first-level headings. </li>
+  <li>Use sentence case for headings. For example, **Extend kubectl with plugins** </li></ul></td><td><ul><li>Use first level heading in blog posts, as Hugo will automatically convert the title section of your post’s front matter into a first-level heading.</li>
+  <li>Use headings level 4 through 6, unless it is absolutely necessary. If your content is that detailed, it may need to be broken into separate articles.</li><li>Use title case for headings. For example, **Extend Kubectl With Plugins** </li></ul></td></tr>
```

Comment:
Hi @Rajakavitha1 :)

+1 for 

> Use sentence case for headings. For example, **Extend kubectl with plugins** ...

Again in line 379 , it is mentioned to use title case for headings. That's a mistake, right?

> Use sentence case for headings. For example, **Extend kubectl with plugins**

Reply:
Ahhhh got it.. sorry about this :)